gadopt/free_surface_equation.py

```diff
@@ -23,7 +23,7 @@ def free_surface_term(
     eq: Equation, trial: fd.Argument | fd.ufl.indexed.Indexed | fd.Function
 ) -> fd.Form:
     r"""Free Surface term: u \dot n"""
-    F = -eq.buoyancy_scale * eq.test * fd.dot(eq.u, eq.n) * eq.ds(eq.boundary_id)
```

Comment:
Sorry to be annoying but do you need to rename this in this PR?

Reply:
You are not! I will revert that change.